lbry/lbry/testcase.py

```diff
@@ -207,10 +210,13 @@ def cleanup():
                 pass
 
         self.addCleanup(cleanup)
```

Comment:
Are you sure this is necessary? Files are automatically closed in Python.

Reply:
For tests purposes, I don't see why it would be strictly necessary, since a tempfile won't need reopened after the test finishes, we don't care about system resources as much... But maybe I am missing something? What if CPython implementation changes? Is there a reason to not close the file deterministically?

Reply:
CPython is not removing the garbage collector so files have always been automatically closed and will always be automatically closed into the foreseeable future. Unless the file is not getting cleaned up?

Generally I strongly dislike having code that doesn't do anything. If this code doesn't do anything (because Python cleans up the file automatically) I don't see the point of keeping this technical debt around.

Reply:
The tests fail without the cleanup - the tempfile is deleted before it can be read...

Reply:
What if you change it to `self.file` instead? Then the variable will stick around until after the test is fully done and cleaned up.

Reply:
That works, do you think it's cleaner?